frontend/src/app/aethel/aethel.component.html

```diff
@@ -105,24 +105,7 @@ <h2 class="title is-2" i18n>Æthel</h2>
             </tr>
         </ng-template>
         <ng-template pTemplate="rowexpansion" let-row>
-            @for (sample of row.samples; track $index) {
-            <tr class="expanded-row">
-                <td colspan="4">
-                    @for (phrase of sample.phrases; track phrase.index) {
-                        <span [class.highlight]="phrase.highlight">{{ phrase.display }} </span>
-                    }
-                </td>
-                <td class="button-container" colspan="1">
-                    <a
-                        [routerLink]="getSampleURL(sample.name)"
-                        class="button is-primary"
-                        i18n
-                    >
-                        Go to sample
-                    </a>
-                </td>
-            </tr>
-            }
+            <tr pp-sample-data class="expanded-row" [aethelResult]="row"></tr>
```

Comment:
We need to use a <tr> here because it messes up PrimeNG's table styling otherwise.